### DISCUSSION TITLE
How to remove/disable underlines in the Win10 taskbar buttons? 🤔

### DISCUSSION THREAD

**Discussion by Milincho** [valinet/ExplorerPatcher#3158] [Q&A]
Is there any way to remove/disable/turn off all these underlines in all buttons in the Win10 style taskbar?

![image](https://github.com/valinet/ExplorerPatcher/assets/60118859/8ee60282-f9af-45ca-bb29-c5ece1e53847)

**Comment by pyrates999** [comment]
Those have always been there to indicate the application is open.

**Comment by Milincho** [comment]
I've been using Windows 11 for so long that I don't remember anymore...   😆

They seem totally redundant to me, because you already know the app is open just seeing the text...

Anyway, the real reason I went back to try the Win10 taskbar style is because it has more space and the Win11's  utterly stupid M$ forced "Taskbar Overflow Menu"
![image](https://github.com/valinet/ExplorerPatcher/assets/60118859/b364e6f7-598a-4193-94c8-574ed41c9a58)

which everybody hates, and for good reason.

Is there any way to just disable the "Taskbar Overflow Menu" in Win11 or disable or shorten the buttons underlines in the Win10 one?

**Reply by pyrates999** [reply]
Look at the options under Taskbar in EP Properties application.

**Comment by Milincho** [comment]
That's precisely why I'm asking...

Because there I don't see any option to remove/disable the "Taskbar Overflow Menu"  in the Win11 one:
![image](https://github.com/valinet/ExplorerPatcher/assets/60118859/7f141515-e623-4b9d-a6c1-288354d3b8f6)

Or remove the underlines in the Win10 one:
![image](https://github.com/valinet/ExplorerPatcher/assets/60118859/4b2b8b29-bfba-4572-8834-305013cec6ae)

Is this something that can be done or is it just impossible?

**Reply by pyrates999** [reply]
This has been around since windows 7.  And has been asked before.  It's there because when you combine them without text showing, you have no way of knowing if the application is open or not.

**Comment by Milincho** [comment]
Ah, ok. They really improved that in Win11 making them almost as narrow as points.

But then they made a taskbar that have less space and that horrible "Taskbar Overflow Menu" system...   🤦🏻
